moveit_planners/chomp/chomp_interface/src/chomp_interface.cpp

```diff
@@ -0,0 +1,78 @@
+/*********************************************************************
```

Comment:
i see there is still a ``chomp_interface_ros.cpp`` but I think those files should have been deleted when this file, and others, were created. isn't this duplicate code?